File: server/src/main/java/com/cloudwise/sap/niping/common/vo/MonitorJob.java
package com.cloudwise.sap.niping.common.vo;

import lombok.Data;

@Data
public class MonitorJob {
	public static final int ACTION_STOP = 0;
	public static final int ACTION_START = 1;
	public static final int ACTION_RESTART = 2;
	private String monitorId;
	private String taskId;
	private int interval;
	private int actionType;
	private JobDesc jobDesc;
	private long modifiedTime;
}

File: monitor/src/models/monitorjob.go
package models

type MonitorJob struct {
	Code          	int64 		`json:"code"`
	Message		string		`json:"msg"`
	Data       	struct {
		MonitorId	string		`json:"monitorId"`
		TaskId		string		`json:"taskId"`
		Interval	int		`json:"interval"`
		ActionType	int		`json:"actionType"`
		JobDesc		struct {
			Router		string		`json:"router"`
			RoundTripTimeB	int		`json:"roundTripTimeB"`
			RoundTripTimeL	int		`json:"roundTripTimeL"`
			BandwithB	int		`json:"bandwidthB"`
			BandwithL	int		`json:"bandwidthL"`
			StabilityB	int		`json:"stabilityB"`
			StabilityD	int		`json:"stabilityD"`
			StabilityL	int		`json:"stabilityL"`
			IdleTimeoutD	int		`json:"idleTimeoutD"`
		}	`json:"jobDesc"`
		ModifiedTime	int64		`json:"modifiedTime"`
	}	`json:"data"`
}